test/app/sessionStoreTest.js

```diff
@@ -1,42 +1,35 @@
 /* global describe, it, before, after */
 
 const Brave = require('../lib/brave')
-const Immutable = require('immutable')
 const {urlInput, navigatorBookmarked, navigatorNotBookmarked, pinnedTabsTabs, tabsTabs} = require('../lib/selectors')
 const siteTags = require('../../js/constants/siteTags')
-const siteUtil = require('../../js/state/siteUtil')
+const settings = require('../../js/constants/settings')
 
-describe('sessionStore', function () {
+describe('sessionStore test', function () {
```

Comment:
These tests are failing for me every time ☹️  ...but it looks good in Travis-ci? 🤔